controller/model/posture_response_handlers.go

```diff
@@ -247,3 +250,87 @@ func (handler *PostureResponseHandler) SetSdkInfo(identityId, apiSessionId strin
 		return postureData
 	})
 }
+
+type ServiceWithTimeout struct {
+	Service *Service
+	Timeout int64
+}
+
+func (handler *PostureResponseHandler) GetEndpointStateChangeAffectedServices(timeSinceLastMfa, gracePeriod time.Duration, onWake bool, onUnlock bool) []*ServiceWithTimeout {
+	affectedChecks := map[string]int64{} //check id -> timeout
+	if onWake || onUnlock {
+		queryStr := fmt.Sprintf("%s=true or %s=true", persistence.FieldPostureCheckMfaPromptOnUnlock, persistence.FieldPostureCheckMfaPromptOnWake)
+		query, err := ast.Parse(handler.env.GetStores().PostureCheck, queryStr)
+		if err != nil {
+			pfxlog.Logger().Errorf("error querying for onWake/onUnlock posture checks: %v", err)
```

Comment:
does this fall through and continue intentionally? if so - cool... just wanted to make sure it is on purpose

Reply:
Yea, if we fail to parse the options are to output an error message or panic. Panics are generally not great to raise in the controller after it is running.

Parse failures won't happen unless the underlying data model is changed (fields/symbols/etc. The API tests would catch the error.